src/main/java/Deadline.java

```diff
@@ -0,0 +1,14 @@
+public class Deadline extends Task {
+    protected String description;
+    protected String by;
```

Comment:
The naming of String by seems a little ambiguous, so it would be good to use a more descriptive variable

Reply:
Oh yes, thank you, that is a good point! I've made changes accordingly.